packages/smooth_app/lib/pages/crop_page.dart

```diff
@@ -227,7 +228,7 @@ class _CropPageState extends State<CropPage> {
   /// Returns a small file with the cropped image, for the transient image.
   ///
   /// Here we use BMP format as it's faster to encode.
-  Future<File> _getCroppedImageFile(
+  Future<File?> _getCroppedImageFile(
```

Comment:
How come? You still return a `File`, and sometimes there's an exception.
You never return `null`, do you?

Reply:
Thats was with the previous implementation, I will revert this.
Thanks